lib/naver/searchad/api/adgroup/service.rb

```diff
@@ -5,35 +5,55 @@ module Adgroup
         class Service < Naver::Searchad::Api::Core::BaseService
 
           def initialize
-            super('https://api.naver.com/', 'ncc/adgroups/')
+            super('https://api.naver.com/', 'ncc/')
```

Comment:
reason why I put them to constants in spec, I have to reuse them across many test cases. I don't think it is necessary here 👍 thanks for your comment anyway